tests/integration/test_nitrate.py

```diff
@@ -96,11 +96,20 @@ def test_existing_dryrun(self):
         self.runner_output = runner.invoke(
             tmt.cli.main,
             ["test", "export", "--nitrate", "--debug", "--dry", "--general",
-             "--bugzilla", "."],
+             "--bugzilla", "--release", "."],
```

Comment:
isn't better to create another new testcase, with more less same functionality, but it uses another code path.